_data/doc_categories.yml

```diff
@@ -8,5 +8,5 @@
   href: docs/development
 - title: How To
   href: docs/howto
-- title: Resources
-  href: docs/resources
+- title: Guides
```

Comment:
For the docs category navigation to pick this up, you'll want to update `_config.yml` as well to set a default category of _Guides_ for any document existing under `_docs/guides`.